src/app/project/FeaturesGrid.js

```diff
@@ -69,12 +74,14 @@ define([
                     renderExpando: true
                 },
                 action: 'Action',
-                subType: 'Treatment/Type'
+                subType: 'Treatment/Type',
+                opacity: {
+                    label: 'Opacity',
```

Comment:
Something like this?
![image](https://cloud.githubusercontent.com/assets/1326248/8440365/fed40494-1f2e-11e5-87c2-c06d852362a3.png)